doc/docs/master/deploy-manage/manage/upgrades.md

```diff
@@ -126,54 +116,62 @@ shows all pods as `READY`:
 
       The `pachd` and `pachctl` versions must both match the new version.
 
-## Troubleshoot upgrades
+## Pachyderm Configuration Values
 
-Most of the issues you might run into when
-upgrading Pachyderm are related to the failure to retrieve 
-then re-set default values that were generated during the first installation.
+### Values Used By Pachyderm At Installation/Upgrade Time
 
-Here is a list of all the values used by Pachyderm when installing/upgrading a cluster. Some are optional and dependent on your installation choices.
+In the following table, you will find a list of all the values used by Pachyderm when installing/upgrading a cluster. Some are optional and dependent on your installation choices.
 
-In general, these values can be provided in three different ways:
+In general, these values can be provided to Pachyderm in two different ways:
 
-- **A - (RECOMMENDED in production) Create your secret(s) ahead of your cluster creation** 
+- **A - (RECOMMENDED in production) Create your secret(s) ahead of time** 
       
       **Create your secret(s) ahead of your cluster creation** and provide each secret name in your values.yaml at the time of the first installation. 
 
-      Find the list of Secret Keys for each setting and their corresponding secret name field in your values.yaml (Column A) in the table below.
+      For each set, find the secret name field that references your secret in your values.yaml (Column A) and its corresponding Secret Key (First column) in the table below.
```

Comment:
I think realistically, we'll have to walk users through the full lifecycle or Pachyderm in each of these sections.  So for this, I think we have to do something like:

```markdown
You can do this step by step, or do it all at once.  Or change it later.  Whatever.

#### Activate Enterprise
Create the following secret in your secret manager:
kind: Secret
metadata:
   name: pachyderm-license
stringData:
    enterprise-license-key: <the text your account rep emailed you, not base64>
Edit values.yaml so that `pachd.license.exsitingSecret` is set to <whatever, not sure>.  Then run "helm upgrade".

#### Enable auth
First, generate a root password with openssl.  Then, add it to this secret.  Then, edit the values.  Next, figure out what IDP you're using.  Then write a dex config for it.  Put the config in this secret.  Edit the values.  Run helm upgrade.


#### Set cluster roles
etc.
```